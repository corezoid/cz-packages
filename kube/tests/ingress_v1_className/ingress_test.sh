#!/usr/bin/env bash
$CZ helm template
