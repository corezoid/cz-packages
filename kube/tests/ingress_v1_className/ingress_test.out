---
# Source: kube_release/charts/kube/templates/ingress_v1.yaml
apiVersion: "networking.k8s.io/v1"
kind: Ingress
metadata:
  name: "foo-ingress"
  labels:
    fooLbl: "fooLblVal"
  annotations:
    fooAnn: "fooAnnVal"
spec:
  rules:
    - host: "foo.example.com"
      http:
        paths:
          - path: "/foo/path"
            pathType: "thePathType"
            backend:
              service: 
                name: "foo_service"
                port: 
                  name: "thePortName"
                  number: 1111
---
# Source: kube_release/charts/kube/templates/ingress_v1.yaml
apiVersion: "networking.k8s.io/v1"
kind: Ingress
metadata:
  name: "ololo-ingress"
spec:
  rules:
    - host: "ololo.example.com"
      http:
        paths:
          - path: "/foo/path"
            pathType: "thePathType"
            backend:
              service: 
                name: "ololo_service"
                port: 
                  name: "thePortName"
                  number: 3333
